lcs/lcs.go

```diff
@@ -0,0 +1,131 @@
+package lcs
+
+import (
+	"strings"
+)
+
+// Options for LCS calculation
+type Options struct {
+	MaxLength     int     // Maximum character length to process (default: 150)
+	CaseSensitive bool    // Whether to consider case (default: false)
+	Threshold     float64 // Default similarity threshold (default: 0.7)
+	WordBased     bool    // Whether to use word-based comparison (default: true)
```

Comment:
Don't mention defaults here, since a zero struct will have zero values which will be different than the defaults
```suggestion
	MaxLength     int     // Maximum character length to process
	CaseSensitive bool    // Whether to consider case
	Threshold     float64 // Default similarity threshold
	WordBased     bool    // Whether to use word-based comparison
```

Reply:
Addressed